app/components/job-target-url.hbs

```diff
@@ -1 +1,4 @@
-{{this.url}}
+<span
+  {{did-insert this.load}}>
```

Comment:
Hmm, not related I think, but I was trying to solve another deprecation that I still can't find. This was an attempt, and seemed like a good idea so I kept it.

Is there a better approach than using the constructor or render-modifiers? Should I revert back to the constructor?

Reply:
I would revert the commit, yes. render-modifiers were a stopgap solution to make transitioning from classic components to Glimmer components easier. The only useful one is `did-update` since Glimmer components don't have a native equivalent hook, but even for that one there are better solutions depending on the use-case.

I don't see a reason to use `did-insert` over the constructor if you don't make use of the dom element.

Reply:
> Hmm, not related I think, but I was trying to solve another deprecation that I still can't find.

What does the deprecation message say?

Reply:
The deprecations are gone now. Probably because of the ember-data bump from 4.7 to 4.8, like you described earlier.
Reverted back the commit about the render-modifiers.